generate_test_reports.sh

```diff
@@ -1,13 +1,12 @@
 #!/bin/sh
 
 # install reportgenerator if not already installed
-dotnet tool install --global dotnet-reportgenerator-globaltool --version 5.1.10 || exit 0
+dotnet tool install --global dotnet-reportgenerator-globaltool --version 5.1.10 || true # exit 0 will kill the script, not what we want
```

Comment:
👌 for Makefiles we should use `exit 0` since it runs one command at a time, but since this is a script itself, I can see the change needing to happen.